base/shared/src/main/scala/scalaz/typeclass/AbelianGroupInstances.scala

```diff
@@ -0,0 +1,12 @@
+package scalaz
+package typeclass
+
+trait AbelianGroupInstances {
+
+  implicit val int: AbelianGroup[Int] = new AbelianGroupClass[Int] {
```

Comment:
So if you have this here it's probably going to clash with `GroupInstances#int` if you ask for `Group[Int]`. You should (assuming we intend to provide a `Group[Int]` at all) keep this and remove the other.

Reply:
yeah, I agree with your above comment, these instances belong to examples, will move them